units/crons.go

```diff
@@ -14,6 +14,8 @@ import (
 	"github.com/mongodb/grip/message"
 	"github.com/mongodb/grip/sometimes"
 	"github.com/pkg/errors"
+	mgo "gopkg.in/mgo.v2"
```

Comment:
there's a helper in anser for is not found. look for adb.ResultNotFound I think..